packages/backend-defaults/src/entrypoints/cache/types.ts

```diff
@@ -44,3 +45,7 @@ export type CacheManagerOptions = {
 export interface PluginCacheManager {
   getClient(options?: CacheServiceOptions): CacheService;
 }
+
+export function ttlToMilliseconds(ttl: number | HumanDuration): number {
```

Comment:
Might be worth letting this accept and return `undefined` too? that's how it's ending up used anyway